tests/dragonfly/cluster_test.py

```diff
@@ -1104,12 +1116,12 @@ async def list_counter(key, client: aioredis.RedisCluster):
     for counter in counters:
         counter.cancel()
 
-    # Push new config
-    await push_config(json.dumps(await generate_config()), [node.admin_client for node in nodes])
-
     # Generate capture, capture ignores counter keys
     capture = await seeder.capture()
```

Comment:
Acually I think we should call capture before we issue SLOT-MIGRATION-FINALIZE

Reply:
We can capture before and after, consistency shouldn't break for outside observers. We could even capture 3 times and check it's the same every time